app/Http/Controllers/Event/EventController.php

```diff
@@ -57,7 +57,9 @@ public function index($organization_id)
 	 */
 	public function show($id)
 	{
-        $event = Event::findOrFail($id);
+    		$event = Event::findOrFail($id);
+		$event->organizationName = $event->organization()->name;
+		$event->organization_id = $event->organization()->id;
 		$creator = null;
```

Comment:
you can retrieve them directly in the view. No need to add more attributes to $event